prov/efa/src/efa_hmem.c

```diff
@@ -86,6 +86,17 @@ static int efa_hmem_support_status_update_cuda(struct efa_hmem_support_status *c
 	}
 
 	cuda_status->p2p_supported = true;
+	/* Eager and runting read protocols */
+	if (rxr_env.efa_max_medium_msg_size != rxr_env_ctime_defaults.efa_max_medium_msg_size) {
```

Comment:
What if users happen to set the same value with the ctime_defaults? We should fail in this case too. You can check the ret of `fi_param_get_**`. If user does not set it, it will return `ENODATA`

Reply:
done.